iothub_client/samples/iothub_client_sample_upload_to_blob_custom/iothub_client_sample_upload_to_blob_custom.c

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// CAVEAT: This sample is to demonstrate azure IoT client concepts only and is not a guide design principles or style
+// Checking of return codes and error values shall be omitted for brevity.  Please practice sound engineering practices
+// when writing production code.
+
+#ifndef DONT_USE_UPLOADTOBLOB
```

Comment:
why? why not control this in the cmake file?

Reply:
That is an enhancement suggestion. But I will work to address it in this PR

Reply:
Actually there is a control already in CMakeLists.txt. I'm removing this #ifndef